lib/src/polygon_clipping/rounder.dart

```diff
@@ -0,0 +1,83 @@
+import 'dart:collection';
+import 'dart:math';
+
```

Comment:
we already have rounding functions in our helpers file.

Please take a look if that works for you: https://github.com/dartclub/turf_dart/blob/f377e468c283bb36d88520016e8e5eb6a975ab1b/lib/src/helpers.dart#L81

Another thing: Please always `Position` class for coordinates (and maybe an extension or extend to add new things)

Reply:
Is it worth running a benchmark on the two methods or should I just use the existing one in helpers?

Reply:
If the existing one works for this use case, let's use that one. And if you see a need for perf. optimization, you can go for it and propose a rewrite of the round we have